### ISSUE TITLE
[Bug]: Error when upgrading to 2.4.0 on BigQuery

### ISSUE THREAD

**Issue by jdewet03** [brooklyn-data/dbt_artifacts#343]
## Overview

We're unable to successfully run` dbt run --select dbt_artifacts `when upgrading to 2.4.0

on-run-end failed, error:
The query is too large. The maximum standard SQL query length is 1024.00K characters, including comments and white space characters.

## How to reproduce

On dbt solution using BigQuery:
- Implement and run dbt_artifacts version 2.3.0
- Upgrade dbt_artifacts version to 2.4.0
- Run:  `dbt run --select dbt_artifacts `

We also tried deleting our dbt_artifacts schemas but same error

## Expected behaviour

We expect to see the dbt command above complete succesfully

## Screenshots

<img width="1144" alt="image" src="https://github.com/brooklyn-data/dbt_artifacts/assets/131745528/f6a98c26-c99c-48cb-8b2f-2a98917f8939">

## Environment

Results of running `dbt --version`: 

```sh
1.4.6
```

Please paste the contents of your `packages.yml` file here:

```yml
packages:
  - package: dbt-labs/audit_helper
    version: [">=0.5.0", "<1.0.0"]

  - package: dbt-labs/dbt_utils
    version: [">=1.0.0", "<2.0.0"]
    
  - package: dbt-labs/codegen
    version: [">=0.5.0", "<1.0.0"]

  - package: data-mie/dbt_profiler
    version: [">=0.5.0", "<1.0.0"]

  - package: dbt-labs/dbt_project_evaluator
    version: [">=0.2.0", "<1.0.0"]

  - package: calogica/dbt_date
    version: [">=0.7.0", "<1.0.0"]

  - package: brooklyn-data/dbt_artifacts
    version: [">=2.3.0", "<3.0.0"]
```

**Comment by glsdown** [comment]
Hi @jdewet03 . Thanks for raising and sorry you are seeing this error. We are not able to replicate it locally, so I need to ask a few follow up questions to work out the cause. How large is your project? Specifically how many models do you have?

**Comment by jdewet03** [comment]
> 

We have 175 models

**Comment by jdewet03** [comment]
Additional context and exact sequence of events:

- We were running successfully on 2.3.0 and once we upgraded to 2.4.0 this is the error we got

```
13:04:01  on-run-end failed, error:
13:04:01   Invalid input: Input number: 1685019801.1016695 cannot round-trip through string representation; error in PARSE_JSON expression
```

- I then ran dbt build --select package:dbt_artifacts manually on prod, without changing anything, to ensure it was not a once-off issue. This time it created a different error

```
13:08:30  on-run-end failed, error:
13:08:30   The query is too large. The maximum standard SQL query length is 1024.00K characters, including comments and white space characters.
```

- I then reverted to 2.3.0 and got yet another new error message

```
13:34:53  on-run-end failed, error:
13:34:53   Inserted row has wrong column count; Has 16, expected 17 at [9:9]
```

- I attempted to run a full refresh: dbt run -f -s dbt_artifacts but this did not change anything

```
13:39:08  on-run-end failed, error:
13:39:08   Inserted row has wrong column count; Has 16, expected 17 at [9:9]
```

- I deleted the dbt_artifact schemas entirely from our database and ran the package again with version 2.3.0 and it was successful

- Lasty, I deleted dbt_artifact schemas entirely from our database and ran the package again with version 2.4.0 and this was the error

```
13:43:52  on-run-end failed, error:
13:43:52   The query is too large. The maximum standard SQL query length is 1024.00K characters, including comments and white space characters.
```

**Comment by mbrummerstedt** [comment]
I have started to see the same issue

**Comment by jdewet03** [comment]
Issue resolved on our side thank you very much

<img width="1134" alt="image" src="https://github.com/brooklyn-data/dbt_artifacts/assets/131745528/6fab196d-4aa4-4301-8f90-b6f568c46fb3">

**Comment by jared-rimmer** [comment]
Closing as should be fixed by `2.4.2`. Feel free to open another issue if the issue persists.
